assets.go

```diff
@@ -157,6 +169,8 @@ type Region struct {
 	width, height float32
 }
 
+// NewRegion Creates  new region from
```

Comment:
I'll have to catch up with the new commits and then reapply some things. Thanks for spotting